pandevice/base.py

```diff
@@ -2047,15 +2215,17 @@ class ParamPath(object):
             enforced in any way from the user's perspective when setting
             parameters, but these values are referenced when parsing any XML
             returned from a live device.
+        references (list): PanObject subclasses that this parameter can reference
 
     """
     def __init__(self, param, path=None, vartype=None,
-                 condition=None, values=None):
+                 condition=None, values=None, references=None):
         self.param = param
         self.path = path
         self.vartype = vartype
         self.condition = condition or {}
-        self.values = values or []
+        self.values = values or ()
```

Comment:
Nope, no reason for it to be mutable.  So just update the params that have `values` to be tuples and change the docstring from saying `values` is a `list` to it being a `tuple`.